src/modules/layout/components/header/user-menu.tsx

```diff
@@ -49,19 +50,19 @@ export function UserMenu() {
         sx={{ mt: 1 }}
       >
         {authStatus === 'authenticated' ? (
-          <>
+          <Box>
             <MenuItem onClick={() => navigate('/account')}>My Account</MenuItem>
             <MenuItem onClick={() => navigate('/wishlist')}>My Wishlist</MenuItem>
             <MenuItem onClick={() => navigate('/accommodations')}>My Accommodations</MenuItem>
             <MenuItem onClick={() => navigate('/bookings')}>My Bookings</MenuItem>
             <MenuItem onClick={handleLogout}>
               <Button fullWidth={true} color="error">
-                Logout
+                Log out
               </Button>
             </MenuItem>
-          </>
+          </Box>
         ) : (
-          <>
+          <Box>
```

Comment:
Why box?

Reply:
> Why box?

Because of this error:
![image](https://github.com/user-attachments/assets/ce639eba-1ad8-45ca-a6f1-1e2bfe3c8f35)